packages/website/docs/TextArea.md

```diff
@@ -22,11 +22,11 @@ If the input can be left blank, the tag “(Optional)” must appear next to the
 
 ## Variants
 
-| **Type**      | **Purpose** |
-| --------- | --------- |
-| **Fixed size** | By default, the size of the text area input box is fixed. A scrollbar automatically appears if text overflows the alloted space. |
+| **Type**           | **Purpose**                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
```suggestion
| - | - |
```

Although I already see it's common within the project, if we do the suggestion we don't have to keep it updated and it'll adjust when we make changes.  Having it all line up raw is nice but the actual content is all on one single line and not very readable as it is.  It's just one more thing to maintain that seems unnecessary, for instance [Toast.md](https://raw.githubusercontent.com/coveo/plasma/master/packages/website/docs/Toast.md) is not aligned anymore, but it never had to be aligned in the first place.  

(Plus I think it looks better than 100+ dashes).

Reply:
It's indeed from the linter, I had to test the script to make sure it's still working with pnpm 7 🙈

Reply:
Okay.  I still like the idea of do it once, forget and not require continuous line changes but if it's automated well then my maintenance point is no longer relevant

Reply:
Yeah I hope we won't have to change it again 🤞 😅 
Feels like .md is design by robots for robots 😆